cmd/xdpcap/main.go

```diff
@@ -39,14 +39,15 @@ func main() {
 		perCPUBuffer = flag.Int("buffer", 8192, "Per CPU perf buffer size to create (`bytes`)")
 		watermark    = flag.Int("watermark", 4096, "Perf watermark (`bytes`)")
 		quiet        = flag.Bool("q", false, "Don't print statistics")
+		flush        = flag.Bool("flush", false, "Flush pcap data written to <output> for every packet received")
```

Comment:
Probably not required?

Reply:
Probably not super useful, but it could help with other special files - ie sending things to `/dev/stderr` or other horrors